.github/workflows/test-mmctl.yml

```
@@ -0,0 +1,50 @@
+name: Test mmctl
```

To keep naming consistent, we can use the following convention:

```suggestion
name: mmctl CI
```